### ISSUE TITLE
Github still used?

### ISSUE THREAD

**Issue by h1z1** [justintv/Twitch-API#686]
Is github still used?  Last commit purged everything yet [your forums](https://discuss.dev.twitch.tv/t/welcome-to-the-twitch-developer-community-read-this-first/2) still reference at least the Issues on github as being used.

Your IRC channel has been a ghost town for months.

 I tried to login with my github id to but it fails.   Github shows the token was generated and is associated with the account yet I get an error:

> "Sorry, there was an error authorizing your account. Perhaps you did 
> not approve authorization? "

Really gotta wonder why on earth you'd pull everything in house ...  when threads are closed after literally 60 minutes of inactivity!

> system 2017-05-26 20:48:43 UTC #2
> 
>    This topic was automatically closed 60 minutes after the last reply. New replies are no longer  allowed.
> 

will remain a mystery I s'pose..

**Comment by 3ventic** [comment]
Github isn't used for documentation anymore, it has moved to
dev.twitch.tv/docs. The IRC channel is practically dead. The forums are
pretty active and the automatic closing time on threads depends on the
category the thread is in. There's a TwitchDev community on the Twitch app
if you're looking for active real time chatting about the API:
https://invite.twitch.tv/p3qzAx

**Comment by h1z1** [comment]
Indeed.   No offense but twitch logs wayyyy too much..  You're Google level creepy.    One of the reasons sites like Github were created was to address the very problem vendors have with running their own publicly facing systems - control.  

I created this because github is still referenced on your forms.
